JUDGEINFO.md

```diff
@@ -0,0 +1,61 @@
+# To the judges
```

Comment:
Now _this_ is what I call proper documentation. Nice work!

Reply:
Thank you.